syntax/javascript.vim

```diff
@@ -324,8 +324,8 @@ if version >= 508 || !exists("did_javascript_syn_inits")
   HiLink jsModuleKeywords       Include
   HiLink jsModuleOperators      Include
   HiLink jsModuleDefault        Include
-  HiLink jsDecorator            Special
-  HiLink jsDecoratorFunction    Special
+  HiLink jsDecorator            Tag
+  HiLink jsDecoratorFunction    Tag
```

Comment:
Not sure `Tag` is the right one for this? The vim help states the following for `Tag`:

```
	 Tag		you can use CTRL-] on this
```